framework/src/media/audio/audio_manager.c

```diff
@@ -922,10 +1003,12 @@ int start_audio_stream_in(void *data, unsigned int frames)
 
 int start_audio_stream_out(void *data, unsigned int frames)
 {
-	int ret;
+	int ret = 0;
 	unsigned int resampled_frames = 0;
 	int prepare_retry = AUDIO_STREAM_RETRY_COUNT;
 	audio_card_info_t *card;
+	float rechanneling_ratio;
```

Comment:
may i know why this is not intialized to zero as others

Reply:
Modified.